src/pages/homepage/weathercountries.js

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import Country from './weathercountry';
+import { getCountries } from '../../redux/countriesState/countriesState';
+import './weathercountries.css';
+
+const Countries = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getCountries());
+  }, [dispatch]);
+  const countries = useSelector((state) => (state.countries));
+  return (
+    <div
+      className="countries-section"
+      data-testid="countries-wrapper"
+    >
+      <div
+        className="header-section"
+      >
+        <h1 className="title-header">World Weather</h1>
+        <p className="choose-country">Choose a country</p>
+        <div className="country-list-header">
+          <h2 className="country-list-title">Countries</h2>
+        </div>
+      </div>
+      <div className="country-card-container">
+        {Object.values(countries).map((country) => (
+          <Country
+            key={country.iso3}
+            name={country.name}
```

Comment:
[REQUIRED] Try to focus more on the mobile screen size Kindly make sure that you have two columns for items without space at the top, bottom, left or right just like the design. The boxes should be touching each other

![image](https://user-images.githubusercontent.com/62938735/213675315-3b2d4da6-c067-4716-9740-eb48e0af31a6.png)